app/components/candidate_interface/complete_section_component.rb

```diff
@@ -1,18 +1,21 @@
 module CandidateInterface
   class CompleteSectionComponent < ViewComponent::Base
-    def initialize(application_form:, path:, request_method:, field_name:, section_review: false)
-      @application_form = application_form
+    attr_reader :section_complete_form, :path, :request_method, :summary_component, :complete_hint_text, :section_review
+
+    def initialize(section_complete_form:, path:, request_method:, summary_component: nil, section_review: false, complete_hint_text: false)
+      @section_complete_form = section_complete_form
       @path = path
       @request_method = request_method
-      @field_name = field_name
+      @summary_component = summary_component
```

Comment:
I've questioned above whether or not this is necessary, but I like the choice of naming here. Leaving this comment as a reminder—if we do decided to keep this dependency, we should either:

- Card up renaming the various `_ReviewComponent` objects that we pass in, making them `_SummaryComponent` classes instead.
- Change this argument to `review_component` for now to keep things consistent.

Reply:
mmm ... good spot.

Yep we are only passing in a SummaryCardComponent once so I think it makes sense to rename the arg to `review_component`